src/requestor/arborist.py

```diff
@@ -83,15 +83,48 @@ def get_resource_paths_for_policy(expanded_policies: list, policy_id: str) -> li
     return []
 
 
-def get_auto_policy_id_for_resource_path(resource_path: str) -> str:
-    resources = resource_path.split("/")
-    policy_id = ".".join(resources[1:]) + "_accessor"
-    return policy_id
+def get_auto_policy_id(
+    resource_path: str = None,
+    resource_paths: list[str] = None,
+    role_ids: list[str] = ["accessor"],
```

Comment:
Would it be more readable to have the default (["accessor"]) specified in the method definition instead of in the method body? (formerly line 124)

Reply:
imo it's more readable in the method definition - but it's better practice to set defaults in the body. Example:
```
def f(list_param = []):
    for item in list_param:
        print(item)
```
if you call `f()` everything goes as expected, but if you call `f(None)`, since you did provide the `list_param` parameter, the default is not used. You end up trying to iterate over `None` and the program crashes.
So this is recommended:
```
def f(list_param = None):
    if not list_param:
        list_param = []
    for item in list_param:
        print(item)
```

Reply:
Yes, sounds good. I'd like to revert back to setting the default in the body. It is a small change that will help prevent crashes.